core/templates/pages/learner-group-pages/edit-group/edit-learner-group-page.component.css

```diff
@@ -0,0 +1,91 @@
+/*
+  NOTE to developers: We generate RTL CSS from this file in the webpack
+  compilation, here are sme additional rules that can be added to the CSS files:
+  https://rtlcss.com/learn/usage-guide/control-directives .
+*/
```

Comment:
Is this true? I thought you said the rtl css is not generated/linked? If not, remove this comment. Ditto elsewhere.

Reply:
Yeah, rtl css is not generated for these files. Have removed this comment from the css files. Thanks!